components/script/dom/shadowroot.rs

```diff
@@ -464,6 +464,7 @@ impl ShadowRootMethods<crate::DomTypeHolder> for ShadowRoot {
         // algorithm steps with this and true.
         self.upcast::<Node>()
             .fragment_serialization_algorithm(true, can_gc)
+            .unwrap() // The spec doesn't mark `innerHTML` as `Throws`.
```

Comment:
I think that even though the specification doesn't mark this method as throwing an exception, we should never crash either. Let's just try to handle this as gracefully as possible.

Reply:
Throws is not an attribute that exists in the spec, so its absence us but meaningful. Adding it in our webidl files is fine.